frontend/src/metabase/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { RoutelessFullPageModal } from "metabase/components/Modal/RoutelessFullPageModal";
+import {
+  WindowModal,
+  WindowModalProps,
+} from "metabase/components/Modal/WindowModal";
+import { FullPageModalProps } from "metabase/components/Modal/FullPageModal";
```

Comment:
I'm a big fan of using `import type` wherever possible. It makes reading the imports a lot easier.

Reply:
Ah I didn't know we even used that! I've always been a fan of type imports as well